moderntimeline.dtx

```diff
@@ -228,6 +228,21 @@
 % the beginning of the year, so is not advisable to use months in dates.
 % Default.
 %
+% \DescribeMacro{\tlenablemarksmo}
```

Comment:
Could you add package options (as a boolean) for this as well?

See l. 357 and 358 for examples of how to declare them.